### ISSUE TITLE
Produced PHA files are not compatible with XSPEC

### ISSUE THREAD

**Issue by giacomov** [giacomov/3ML#94]
A .pha file produced with .write_pha of the OGIPLike plugin is not readable by XSPEC, because it is missing a column.

Furthermore, its GROUPING is not recognized.

Here is the XSPEC error:

```
XSPEC12>data test_out.pha{1}
***Warning: Unrecognized grouping for channel(s). It/they will be reset to 1.

***XSPEC Error:  data file missing required entries: ANCRFILE
```

Here is the code to reproduce the problem, using the data in attachment:

```python
from threeML import *

dd = '100724029/'

lle = OGIPLike("LLE",dd+"bn100724029_LAT-LLE_srcspectra.pha{1}",
                        dd+"bn100724029_LAT-LLE_bkgspectra.bak{1}",
                        dd+"bn100724029_LAT-LLE_weightedrsp.rsp{1}")
lle.set_active_measurements('3e4-1e7')

lle.rebin_on_background(5.0)

lle.write_pha("test_out.pha", overwrite=True)
```

A probably unrelated problem is produced by this similar code, using one of the .pha files in the examples:

```python
from threeML import *

# Update this with your path
dd = '/home/giacomov/develop/3ML/examples/'

lle = OGIPLike("LLE",dd+"bn090217206_n9_srcspectra.pha{1}",
                        dd+"bn090217206_n9_bkgspectra.bak{1}",
                        dd+"bn090217206_n9_weightedrsp.rsp{1}")
lle.set_active_measurements('10.0-1000.0')

lle.rebin_on_background(5.0)

lle.write_pha("test_out.pha", overwrite=True)
```

and this is the XSPEC error:

```
XSPEC12>data test_out.pha{1}

***XSPEC Error:  data file missing required entries: 
Detector channel numbers missing in extension: SPECTRUM

***XSPEC Error:  data file missing required entries: ../test_out.pha

***XSPEC Error: Error reading data: File ../test_out.pha
```

[data_files.tar.gz](https://github.com/giacomov/3ML/files/544077/data_files.tar.gz)

**Comment by drJfunk** [comment]
@giacomov I'm on this. Good catch! What time scale do you need this on? I'm busy until this afternoon EU time.

**Comment by giacomov** [comment]
no worries, no rush on my side. I'm going to be offline (traveling) the whole weekend.

**Comment by drJfunk** [comment]
fixed with #95
